app/src/main/AndroidManifest.xml

```diff
@@ -738,6 +738,18 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".component.ui.overview.MainActivity" />
         </activity>
+        <activity
+            android:name=".component.ui.ticket.TicketScanActivity"
+            android:configChanges="orientation|screenSize"
+            android:label="@string/app_name"
+            android:launchMode="singleTop"
+            android:parentActivityName=".component.ui.ticket.TicketScanActivity"
+            android:theme="@style/AppThemeFullscreen">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value="de.tum.in.tumcampusapp.component.ui.ticket.TicketScanActivity" />
+        </activity>
```

Comment:
When the device is rotated, a grey sidebar appears:
<img width="468" alt="bildschirmfoto 2018-06-12 um 17 10 35" src="https://user-images.githubusercontent.com/30290755/41299890-c3fb90ca-6e64-11e8-9c47-3a18c1c4a225.png">
Does this also appear on a real device? If so, I think we could lock the screen to portrait mode for the validity scan.